test/saithrift/tests/sail2.py

```diff
@@ -20,6 +20,27 @@
 import sai_base_test
 
 @group('l2')
+class UnknownUnicastTest(sai_base_test.ThriftInterfaceDataPlane):
+    def runTest(self):
```

Comment:
this test need to be applied on vlan. i've just checked the previous BcastUcastTest and i think this test can be appended at the previous test.
try:
send packet with the dst mac not in the FDB table
verify no ports received that packet
check the counters_egress_drop == 1